README.md

```diff
@@ -57,20 +57,24 @@ The reason we are using this is it's very easy to customize, highly flexible and
 - Modules: Added `modules` folder where each page is a module e.g. `modules/about-us`. Create components inside the module.
 
   E.g. `about-us/OurMission`
-    ```
-    modules
-      about-us
-        OurMission
-    ```
+
+  ```
+  modules
+    about-us
+      OurMission
+  ```
+
 - The rest of the common components such as `<PageHeader />` and `<MainNav />` should be placed in the `components` folder. If there are multiple components related to each other create a folder for it.
 
-    E.g. `Card/CardHeader`
-    ```
-    Card
-        CardHeader
-        CardTitle
-        CardFooter
-    ```
+  E.g. `Card/CardHeader`
+
+  ```
+  Card
+      CardHeader
+      CardTitle
+      CardFooter
```

Comment:
@sonam-serchan  - FYI - Just a question, nothing about changes.

Shouldn't it be this ?
- Card
          Title
         Footer
.......

Because it is already under the card folder.

Reply:
Yeah we can name the files accordingly!

However, the component name is better to be the same way - CardHeader, CardTitle etc.